toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -486,8 +486,13 @@ build_rpm_in_chroot_no_install pam
 # libselinux requires libsepol
 chroot_and_install_rpms libsepol
 build_rpm_in_chroot_no_install libselinux
-# util-linux and rpm require libselinux
+
+# libcap-ng needs: swig, python3
+build_rpm_in_chroot_no_install libcap-ng
+
```

Comment:
You'll need to following line to ensure the `python3-libcap-ng` package is copied into the output folder, since its name isn't prefixed with `libcap-ng`.

```suggestion
copy_rpm python3-libcap-ng
```